### PR TITLE
Modifies some refinery weapon descriptions

### PR DESCRIPTION
**Pull request by tired-wired**
## About The Pull Request
Modifies fullstop, rats, streetlight, and yun's office weapon descriptions. I plan to edit more of the others, but this will do for feedback and a start.

## Why It's Good For The Game
Fancier descriptions for fun and references to their initial wielders.

## Changelog
:cl:
tweak: tweaked fullstop, rats, streetlight, and yun's office weapon descriptions
/:cl:

### CODE REVIEW

**Review by Kitsunemitsu — APPROVED** [review]
Love these.

File: code/game/objects/items/ego_weapons/non_abnormality/streetlight.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 //They're generic weapons for the refinery, which is fine
 /obj/item/ego_weapon/city/streetlight_greatsword
 	name = "streetlight greatsword"
-	desc = "A greatsword used by the streelight office."
+	desc = "A greatsword used by a fixer with big shoes to fill.'Maybe... I should have told her how I feel.'"
```
Comment (vampirebat74) [inline code/game/objects/items/ego_weapons/non_abnormality/streetlight.dm:5]: How I felt*

Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/non_abnormality/streetlight.dm:5]: Could still feel it, it works either way

Comment (vampirebat74) [reply code/game/objects/items/ego_weapons/non_abnormality/streetlight.dm:5]: "have told her" is past tense. While I'd normally say an argument over semantics is pointless, writing quality is the entire point of this PR

Comment (tired-wired) [reply code/game/objects/items/ego_weapons/non_abnormality/streetlight.dm:5]: That particular line is a direct quote from Library of Ruina, but I see your point! I'll make a note to go back over it in my next batch.

**Pull request merged**